testdata/bundles/digest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "foo",
+  "version": "1.0",
+  "schemaVersion": "99.99",
+  "images": {
+    "my-microservice": {
+      "contentDigest": "sha256:aaaaaaaaaaaa...",
```

Comment:
On the basis that open source tests are a form of documentation, I'd prefer it if we used a proper content digest e.g. "sha256:deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef" (or a vegetarian alternative).

Reply:
Yep, good idea. I took this directly from one of the examples in the spec examples directory.